package.json

```diff
@@ -4,23 +4,27 @@
   "private": true,
   "scripts": {
     "docusaurus": "docusaurus",
-    "start": "docusaurus start",
-    "build": "docusaurus build",
-    "swizzle": "docusaurus swizzle",
-    "deploy": "docusaurus deploy",
-    "clear": "docusaurus clear",
-    "serve": "docusaurus serve",
-    "write-translations": "docusaurus write-translations",
-    "write-heading-ids": "docusaurus write-heading-ids",
+    "start": "npm run graphql:generate && docusaurus start",
+    "build": "npm run graphql:generate && docusaurus build",
+    "swizzle": "npm run graphql:generate && docusaurus swizzle",
+    "deploy": "npm run graphql:generate && docusaurus deploy",
+    "clear": "npm run graphql:generate && docusaurus clear",
+    "serve": "npm run graphql:generate && docusaurus serve",
+    "write-translations": "npm run graphql:generate && docusaurus write-translations",
+    "write-heading-ids": "npm run graphql:generate && docusaurus write-heading-ids",
```

Comment:
do we really have to generate graphql on _every_ command? 😅

Reply:
Yes, I don't think so. I just wanted to hear from you 😅 What about having it on below?

- npm start
- npm build
- npm deploy